content/1.7/back-office/design/image-settings/add-edit-image-setting.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Add new Image type
+weight: 1
+---
+
+# Design > Image Settings > Add new Image type
+
+## Adding or editing Image type UI
+
+![Adding or editing Image type panel](static/img/image-settings-add-edit-image-type.png)
+
+This configuration UI is about adding the new Image type into the Image type listing.
+
+There are the following input text fields, that can be set:
+
+- **Name for the image type**  - this is a required field, that describes the main name of the image type. The image type name should be used with underscores and hyphens only. Fore example, that could be _small_custom_, _large_, _custom-modified_small_. By default this input text field is empty.
+- **Width** - this is a required field, that sets the numeric value of Image type width in pixels. By default, this input text field is empty.
+- **Height** - this is a required field, that sets the numeric value of Image type height in pixels. By default, this input text field is empty.
```

Comment:
1. Tooltip is missing.
2. The error message when entering an invalid  Height is missing "The  Height field is invalid."
3. The number of required characters is missing
4. The error message when trying to save with empty  Height is missing "The  Height field is required."

Reply:
@marwachelly hey, thanks for your feedback, I have made separate paragraph for notification messages, you want to replace those here?

Reply:
@SimonasB88 Thank you for your work. I think it will be better If each description includes all the corresponding notices to avoid the risk of forgetting some (tooltip / max chars...)

Reply:
@marwachelly done :)